logger.go

```diff
@@ -24,6 +25,7 @@ type ILogger interface {
 // ZapCompatibleLogger is a basic logging interface for golog.Logger (alias for *zap.SugaredLogger) and RDK loggers.
 type ZapCompatibleLogger interface {
 	Desugar() *zap.Logger
+	Level() zapcore.Level
```

Comment:
adding Level seems to be the easiest thing to do (since it's already present in the RDK version of this interface), but curious if there are other options.

Reply:
What you did seems good to me.

Reply:
removed since it was unnecessary